client/app/js/jquery.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.6.1 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
Jquery is a big library additional to the current AngularJS codebase and we shall find an alternative to keep the codebase small.

File to be removed from the pull request.

Reply:
Still to be removed.

I'm going to fix this myself while integrating the pull request.